docs/src/glossary_and_notations.rst

```diff
@@ -0,0 +1,60 @@
+.. _glossary_and_notations:
+
+
+===========================
+Glossary and Notations
+===========================
+
+Notations
+---------
+
+.. glossary::
+
+    minus index
```

Comment:
Can you add the definition of `minus index`?

Reply:
What I intended is a section "minus index" with subsections groups and individual variables.

Reply:
That makes sense, I added a crossref from the notaion to this entry in the Glossary.